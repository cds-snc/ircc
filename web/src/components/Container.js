import styled from 'react-emotion'

const Container = styled.div`
  width: ${props => (props.width ? props.width : '100%')};
  margin-bottom: 23px;
  background-color: #fff;
  border: 1px solid #2572b4;
  border-radius: 4px;
  -webkit-box-shadow: 0 1px 1px rgba(0, 0, 0, .05);
  box-shadow: 0 1px 1px rgba(0, 0, 0, .05);
`

export default Container
