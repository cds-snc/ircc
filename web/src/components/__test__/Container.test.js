import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'
import Container from '../Container'

describe('<Container />', () => {
  it('can be rendered', () => {
    let wrapper = shallow(<Container width="1000%" />)
    expect(wrapper.is('div')).toBeTruthy()
  })

  it('accepts a width prop', () => {
    let wrapper = shallow(<Container width="1000%" />)
    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
