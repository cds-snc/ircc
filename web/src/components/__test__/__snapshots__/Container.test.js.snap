// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> accepts a width prop 1`] = `
<div
  className="css-h9q2h6"
  width="1000%"
/>
`;
